etc/fastly.vcl

```diff
@@ -94,6 +94,19 @@ sub vcl_recv {
 }
 
 sub vcl_fetch {
+
+    # Remove Set-Cookies from responses for static content
+    # to match the cookie removal in recv.
+    if (req.url ~ "^/(pub/)?(media|static)/.*") {
+        unset beresp.http.set-cookie;
+
+        # Set a short TTL for 404's
+        if (beresp.status == 404) {
+            set beresp.ttl = 300s;
```

Comment:
Its static assets in magento they are unlikely to be added sooner and the 404 can be purged. This felt like the balance of offloading refreshes of missing assets.